openapi/ga4gh-tool-discovery.yaml

```diff
@@ -673,29 +682,66 @@ definitions:
         type: string
         description: >-
           Optional url to the underlying content, should include version
-          information, and can include a git hash.  Note that this
-          URL should resolve to the raw unwrapped content that would otherwise
-          be available in content. One of url or content is required.
+          information, and can include a git hash.  Note that this URL should
+          resolve to the raw unwrapped content that would otherwise be available
+          in content. One of url or content is required.
         example:
           descriptorfile:
-            url: https://raw.githubusercontent.com/ICGC-TCGA-PanCancer/pcawg_delly_workflow/ea2a5db69bd20a42976838790bc29294df3af02b/delly_docker/Delly.cwl
+            url: >-
+              https://raw.githubusercontent.com/ICGC-TCGA-PanCancer/pcawg_delly_workflow/ea2a5db69bd20a42976838790bc29294df3af02b/delly_docker/Delly.cwl
           containerfile:
-            url: https://raw.githubusercontent.com/ICGC-TCGA-PanCancer/pcawg_delly_workflow/c83478829802b4d36374870843821abe1b625a71/delly_docker/Dockerfile
+            url: >-
+              https://raw.githubusercontent.com/ICGC-TCGA-PanCancer/pcawg_delly_workflow/c83478829802b4d36374870843821abe1b625a71/delly_docker/Dockerfile
   ServiceInfo:
     type: object
-    description: Describes this registry to better allow for mirroring, indexing, and useful information about the running service, including supported versions and
-      default settings.
+    description: >-
+      Describes this registry to better allow for mirroring, indexing, and
+      useful information about the running service, including supported versions
+      and default settings. Partially synced with
+      https://github.com/ga4gh-discovery/service-info
     required:
+      - id
+      - name
       - version
       - api_version
       - contact_info_url
     properties:
+      id:
+        type: string
+        description: >-
+          Unique ID of this service. Reverse domain name notation is
+          recommended, though not required.
+        example: org.ga4gh.service
+      name:
+        type: string
+        description: Name of this specific service.
+        example: 1000 Genomes Project
+      description:
+        type: string
+        description: Description of the service.
+        example: >-
+          The 1000 Genomes Project is the largest public catalogue of human
```

Comment:
British/Canadian spelling of catalog.  :smile:  :+1:

Reply:
This is inherited from 
https://github.com/ga4gh-discovery/service-info/blob/develop/service-info.yaml#L42 

I think TRS can remain international ;)